### DISCUSSION TITLE
Fullscreen support for shader editor

### DISCUSSION THREAD

**Discussion by shahriarlabib000** [godotengine/godot-proposals#10499] [Editor]
I think devices with smaller screens like Android devices will benefit from it.
gdscript editor already has a Fullscreen mode maybe something like that.It will especially help when code line gets long.
here is how much room i get to write code as of 4.3.stable
![Screenshot_2024-08-19-23-41-30-93_653f2d6f0c14415f40b50121f34f510c](https://github.com/user-attachments/assets/070ec2b9-fe35-4e71-a5dc-4a82aa878838)

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/5071

**Reply by shahriarlabib000** [reply]
to clarify, I didn't propose to combine shader editor and gdscript editor. I just suggested to
add a Fullscreen mode to the existing shader editor

**Reply by AThousandShips** [reply]
Thank you for clarifying, I think it falls under the same scope but good to add that as a suggestion over there
